portfolio/src/main/webapp/script.js

```diff
@@ -1,13 +1,13 @@
 // Copyright 2019 Google LLC
 //
-// Licensed under the Apache License, Version 2.0 (the "License");
+// Licensed under the Apache License, Version 2.0 (the 'License');
```

Comment:
nit: best to leave the copyright as is

Reply:
Whoops should have checked my replace all a bit more carefully